aws/table_aws_emr_studio.go

```diff
@@ -0,0 +1,196 @@
+package aws
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/service/emr"
+	"github.com/aws/aws-sdk-go-v2/service/emr/types"
+
+	emrv1 "github.com/aws/aws-sdk-go/service/emr"
+
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableAwsEmrStudio(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_emr_studio",
+		Description: "AWS EMR Studio",
+		Get: &plugin.GetConfig{
+			KeyColumns: plugin.SingleColumn("studio_id"),
+			IgnoreConfig: &plugin.IgnoreConfig{
+				ShouldIgnoreErrorFunc: shouldIgnoreErrors([]string{"InvalidRequestException"}),
+			},
+			Hydrate: getEmrStudio,
+			Tags:    map[string]string{"service": "elasticmapreduce", "action": "DescribeStudio"},
+		},
+		List: &plugin.ListConfig{
+			Hydrate: listEmrStudios,
+			Tags:    map[string]string{"service": "elasticmapreduce", "action": "ListStudios"},
+		},
+		GetMatrixItemFunc: SupportedRegionMatrix(emrv1.EndpointsID),
```

Comment:
Since we're in the process of removing AWS SDK V1 support from the plugin, can we use the endpoint ID from `endpoint_service_ids_gen.go` instead?